### ISSUE TITLE
[med] JAWS+Chrome: Many Pages: Search field name describes input not purpose

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5658]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has recent chats or contacts added.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Tab to focus "+" button and press Enter key.
3. Tab to focus "New Group" element and press Enter key.
4. Tab/Shift+Tab to focus the search field and note the label announced.

## Expected Result:
The label should for fields should describe the purpose and placeholder text used to provide additional descriptive information when necessary. i.e. Field Name: "Search Contacts", Placeholder: "Name, Email, or phone number".

## Actual Result:
There is no visible label for the field (Issue 5182583). The placeholder is text used as the programmatic name as well which announces without any context on the purpose of the field. i.e. "Name, Email, or phone number".

HTML:
`<input placeholder="Name, email, or phone number" autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" spellcheck="true" type="text" class="css-11aywtz r-tz77pi" value="" style="background-color: rgb(255, 255, 255); border-color: rgb(236, 236, 236); border-radius: 8px; border-width: 1px; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; height: 42px; padding: 10px 12px; vertical-align: middle;">`

Other occurrences:
- New Chat panel: search field.

## Workaround:
No workaround

## Area issue was found in:
Create new group overlay, Create new chat overlay

## Failed WCAG checkpoints
2.4.6

## User impact:
Screen reader users will hear the input suggestions from the placeholder text "Name, Email, or phone number" but will not know if the field creates a new group with that input or searches, or something else.

## Suggested resolution:
As suggested in issue 5182583, do not use the placeholder text as the label and make a persistently visible label that describes the purpose of the field e.g. 'Search' (consistent with the Emoji search in chat) or similar. Leave the placeholder text and include aria-placeholder for added compatibility.
https://www.w3.org/WAI/WCAG21/Techniques/general/G131.html
https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#aria-placeholder

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/136068457-db7e53c4-101f-4fcc-9e72-68c50b95dccd.mp4

![Bug5186854_Chrome_Jaws-Create_New_Group-Search_field_name_describes_input_not_purpose](https://user-images.githubusercontent.com/88733897/136068474-2d03903c-39b5-42b8-a3a5-feb937ee7e0e.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
